lib/logitech_receiver/descriptors.py

```diff
@@ -143,7 +143,7 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 # Keyboards
 
 _D('Wireless Keyboard K230', protocol=2.0, wpid='400D')
-_D('Wireless Keyboard K270', protocol=1.0,
+_D('Wireless Keyboard K270', protocol=1.0, wpid='4023',
```

Comment:
I confirmed my Combo came with a K270 HID++ 2.0. Look to my dump at #234 .

So we cannot add `wpid='4023'` since this wpid refers to other HID version. We might add a second entry for K270 HID++ 2.0 and it shall have a different codename.